### DISCUSSION TITLE
"There was an error when trying to allow this organization to assign seats."

### DISCUSSION THREAD

**Discussion by LeoYMCMB** [community/community#144463] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I try to activate GitHub Copilot using Azure as the payment method, I receive the following message: "There was an error when trying to allow this organization to assign seats."

**Comment by M-tech-cmd** [comment]
Check Azure Subscription: Ensure that your Azure subscription is active and has sufficient credits or payment methods set up.

Verify Permissions: Make sure you have the necessary permissions in both GitHub and Azure. [You need to be an owner of the GitHub organization and have admin consent in Azure](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/connecting-an-azure-subscription)[1](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/connecting-an-azure-subscription).

[Update Billing Information: Double-check that your billing information in both GitHub and Azure is correct and up to date](https://github.com/orgs/community/discussions/131625)[2](https://github.com/orgs/community/discussions/131625).

Reconnect Azure Subscription: Try disconnecting and then reconnecting your Azure subscription to your GitHub organization. [This can sometimes resolve connection issues](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/connecting-an-azure-subscription)[1](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/connecting-an-azure-subscription).

Contact Support: If the issue persists, consider reaching out to GitHub support for further assistance. [They can provide more specific guidance based on your account details](https://github.com/orgs/community/discussions/131625)[2](https://github.com/orgs/community/discussions/131625).
